cmd/world/root/create.go

```diff
@@ -22,21 +23,26 @@ const TemplateGitUrl = "https://github.com/Argus-Labs/starter-game-template.git"
 
 // createCmd creates a new World Engine project based on starter-game-template
 // Usage: `world cardinal create [directory_name]`
-var createCmd = &cobra.Command{
-	Use:   "create [directory_name]",
-	Short: "Create a World Engine game shard from scratch",
-	Long: `Create a World Engine game shard based on https://github.com/Argus-Labs/starter-game-template.
+func getCreateCmd(writer io.Writer) *cobra.Command {
+	createCmd := &cobra.Command{
+		Use:   "create [directory_name]",
+		Short: "Create a World Engine game shard from scratch",
+		Long: `Create a World Engine game shard based on https://github.com/Argus-Labs/starter-game-template.
 If [directory_name] is set, it will automatically clone the starter project into that directory. 
 Otherwise, it will prompt you to enter a directory name.`,
-	Args: cobra.MaximumNArgs(1),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		logger.SetDebugMode(cmd)
-		p := tea.NewProgram(NewWorldCreateModel(args))
-		if _, err := p.Run(); err != nil {
-			return err
-		}
-		return nil
-	},
+		Args: cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			logger.SetDebugMode(cmd)
+			p := tea.NewProgram(NewWorldCreateModel(args), tea.WithOutput(writer))
+			if _, err := p.Run(); err != nil {
+				fmt.Println(err)
```

Comment:
Doesn't cobra print errors to the screen? Is this fmt.Println necessary?

Reply:
this error is from bubletea lib not cobra. so yes this is necessary.

Reply:
ahhh, sorry. this is inside cobra. yes it will be printed by cobra.
will remove this line.